### PR TITLE
Generalise gdoc structure for easier reuse

### PR DESCRIPTION
**Pull request by pvhee**
Things to handle on deploy:

- [ ] add the following to .env in Netlify

```
GOOGLE_DOC_LANG="pt"
GOOGLE_DOC_SPREADSHEET_ID="1VT6yvROk2V_Z438YdzBHkjeplqOeaTkAR4SfifBsMqg"
```

### CODE REVIEW

**Comment by pvhee** [comment]
@Quicksaver a start to generalise field names to it is useful to others. I know this is not removing all PT strings, but that seems a lot of work at this phase.

File: gatsby-config.js [hunk 1]
```diff
@@ -112,17 +109,24 @@ module.exports = {
...
-          // NOTE: if the desired filter column has the same name as the
-          // original field, the original will be overwritten; this should be
-          // fine for most cases though.
-          [cur]: node[FILTERS[cur].field].split(',').map(n => n.trim()),
-        }), node),
+        mapNode: node => ({
+          // Add translated field names so we can refer to them in Gatsby
+          ...Object.keys(FIELDS).reduce((acc, cur) => ({
+            ...acc,
+            [cur]: node[FIELDS[cur].field],
```
Comment (Quicksaver) [inline gatsby-config.js:116]: You can just add this line below on the existing reduce, no need for a new reduce.

Comment (pvhee) [reply gatsby-config.js]: doh, fixed, thx :)

**Comment by pvhee** [comment]
@Quicksaver I've changed this slightly to load config from JSON which is certainly better, but I kept the language setting as I think it's important the structure of the doc itself is part of the codebase as GraphQL depends on this. I.e. allowing anyone to freely configure fields will lead to build errors, so I rather have all the available config inside the codebase, but support different languages.

**Comment by pvhee** [comment]
Note that now also by default this codebase will use the generalised (english) structure of the doc: https://docs.google.com/spreadsheets/d/1i6mNf0uqnBUSiebSWB7YUKAdvokQZpGq-o7rFxbc7Gs/edit#gid=0 (which has a copy of the #cap data as of yesterday, 21th of April 2020)

File: gatsby-config.js (hunk 1 above)
Comment (Quicksaver) [reply gatsby-config.js]: Actually nevermind this, I hadn't noticed this was about FIELDS, not FILTERS, a reduce inside another there would just make the inner reduce run needlessly several times. Sorry about the confusion... 🤦  I undo'd your change there as it was my fault to lead you in error.

**Comment by Quicksaver** [comment]
Nice about the json, and yeah I had not realized graphql needs hardcoded fields, makes sense. One thing, `GOOGLE_DOC_SPREADSHEET_SHEET` is missing in *.env.example*, we'll need that also on Netlify to deploy (I only noticed because I tried it locally with the live sheet and it failed).

**Pull request merged**